Tools/dollymix/zslice.py

```
@@ -0,0 +1,65 @@
+base = Image.open(os.path.concat(path,basefile+ext))
+amount = base.height // size
+
+states = []
+
+for i in range(amount):
+    name = basefile+str(i+1)
+    filename = name+ext
+    states.append(name)
+    print(filename)
+    if(reverse):
+        base.crop((0,size*(amount-1-i),base.width,size*(amount-i))).save(os.path.concat(path,filename))
```

Pylint (3.3.7)

[error] Module 'posixpath' has no 'concat' member

(E1101)